server/ai_process.go

```diff
@@ -1533,6 +1533,10 @@ func processAIRequest(ctx context.Context, params aiRequestParams, req interface
 			break
 		}
 
+		if cap == core.Capability_LiveVideoToVideo {
+			ctx, cancel = context.WithTimeout(ctx, 15*time.Second)
+			defer cancel()
```

Comment:
Will this stop payments or anything else long-lived which depends on the context, once this function exits? ([related](https://github.com/livepeer/go-livepeer/pull/3468))

Reply:
Good point. Moved to apply only to the HTTP Req